### ISSUE TITLE
Down load is very slow

### ISSUE THREAD

**Issue by DanielWinners** [We-the-People-civ4col-mod/Mod#380]
Is there anywhere else on the net where I can down load this mod. The file saying it will take two days to down load?

**Comment by raystuttgart** [comment]
Sorry, but we have no other download site.
It has been working fine for all of us so far.

The mod is pretty big though.
It is after all about 1 GB.

Where are you living?
What kind of internect connection do you have?

**Comment by DanielWinners** [comment]
I am in Zimbabwe.. but have a fibre connection of 20mbps, bit surprised its not down loading faster, trying to figre out why its downloading so slowly and if i can do anything to speed it up... Looks like an amazing mod just wish it would download.

**Comment by DanielWinners** [comment]
It worked I left it downloading in the back ground took less than two hours dont know why it said it would take two days?

**Comment by Nightinggale** [comment]
You shouldn't trust estimated download times too much. They are calculated based on the assumption that your speed is stable and won't change. However you might encounter servers, which caps your download to reduce server load. If it detects that you constantly hit the limit, then it will give you a higher limit if it has free capacity. If a server does this, you will see the download starting slow and then stepping up multiple times. The estimated time left will be way too long until it's done figuring out how much server speed to allocate to you.

In short: never trust early download estimates. They can change a lot.

There can also be a lot of other reasons, like internet traffic and overloaded ISP. It's impossible to tell precisely why you encountered issues, but since they are solved, I will close this issue as it's "fixed" (or rather it was never an issue).
